internal/controller/policyserver_controller_deployment.go

```diff
@@ -372,10 +363,6 @@ func (r *PolicyServerReconciler) configureMutualTLS(ctx context.Context, policyS
 		return nil
 	}
 
-	admissionContainer.Env = append(admissionContainer.Env, corev1.EnvVar{
-		Name:  "KUBEWARDEN_CLIENT_CA_FILE",
-		Value: filepath.Join(kubewardenCAVolumePath, constants.CARootCert),
-	})
```

Comment:
Don't we still need to set this environment variable?

Reply:
no we don't, it is configured in the `if` block above alongside the user provided client CA

Reply:
true, it was hidden by GH review :+1: